src/lib/reflect.ts

```diff
@@ -314,6 +315,7 @@ export class ReflectedTypeRef<T extends RtType = RtType> {
     isFalse(): this is ReflectedFalseRef { return this.kind === 'false'; }
     isStringLiteral(): this is ReflectedLiteralRef<string> { return this.kind === 'literal' && typeof this.ref === 'string'; }
     isNumberLiteral(): this is ReflectedLiteralRef<number> { return this.kind === 'literal' && typeof this.ref === 'number'; }
+    isBigIntLiteral(): this is ReflectedLiteralRef<number> { return this.kind === 'literal' && typeof this.ref === 'bigint'; }
```

Comment:
Should this be `ReflectedLiteralRef<bigint>` or `ReflectedLiteralRef<BigInt>` rather than `ReflectedLiteralRef<number>`?

Reply:
Right! i missed that :)